src/tests/electron/common/create-application.ts

```diff
@@ -1,68 +1,61 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 import * as Electron from 'electron';
-import { AppConstructorOptions, Application } from 'spectron';
-import { retry } from 'tests/common/retry';
-
-import {
-    DEFAULT_APP_CONNECT_RETRIES,
-    DEFAULT_APP_CONNECT_TIMEOUT_MS,
-    DEFAULT_CHROMEDRIVER_START_RETRIES,
-    DEFAULT_CHROMEDRIVER_START_TIMEOUT_MS,
-} from 'tests/electron/setup/timeouts';
+import { _electron as electron } from 'playwright';
 import { AppController } from './view-controllers/app-controller';
-
-export interface AppOptions extends Partial<AppConstructorOptions> {
+export interface AppOptions {
     suppressFirstTimeDialog: boolean;
+    env?: {
```

Comment:
Should this be optional? In the code below, we're assuming it exists. If we can safely make that assumption then we should change this or otherwise check in other locations.

Reply:
Yes we rarely pass `env` when calling createApplication (basically only used to set the adb folder to something fake to force a prompt)

Reply:
Sorry, I actually misread it. We are checking to validate the options object.